samples/System.IO.Pipelines.Samples/System.IO.Pipelines.Samples.csproj

```diff
@@ -21,5 +21,6 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.Hosting.Server.Abstractions" Version="1.1.0" />
     <PackageReference Include="Microsoft.AspNetCore.Server.Kestrel" Version="1.1.0" />
+    <PackageReference Include="System.Net.NameResolution" Version="4.4.0-beta-24913-02" />
```

Comment:
This is pretty random. Why does it need to be version 4.4.0-beta-24913-92? Doesn't 4.3.0 work? Also, what is it required for?